osquery/tables/system/cpuid.cpp

```diff
@@ -80,6 +80,7 @@ std::map<size_t, std::vector<FeatureDef>> kCPUFeatures{
      }}};
 
 static inline void cpuid(size_t eax, size_t ecx, int regs[4]) {
+#if defined(__x86_64__)
```

Comment:
Would it make sense to disable this table from the SQLiteDBManager constructor in `osquery/osquery/sql/sqlite_util.cpp` when compiling on Aarch64?

Reply:
I'm not quite sure how to do that. Would it work to just not compile it (don't compile cpuid.cpp on aarch64 and reove cpuid.table from the list)?

Reply:
The cpuid table is currently handled as a platform-independent table, which makes it always present and safe to reference without guarding the operation with something like a discovery query (which makes query packs easier to write).

1. If the intent is to postpone the implementation of this table for later, we could either disable the table from the `SQLiteDBManager` class (with `::setDisabledTables`) or update the row generation to return an empty set of rows and generate a log message.

2. Another approach, if this table will never be implemented on Aarch64, could be to convert it to a platform-specific table.

Reply:
Looking at the cpuid schema, it looks like it won't map very well to AArch64, so we should probably make it platform-specific. It would be easeir to make an arm_cpu_features table or something when I (or someone else) have time to write it.

Reply:
I am not quite familiar with the AArch64 architecture, but if the problem is the schema alone osquery supports a feature named extended schema, that automatically enables/disables columns based on the platform.

If you agree, we can deal with this table in the future and for now just do a `return {};` in the `genCPUID()` function

Reply:
Okay, that makes sense. I've done that and disabled the test on aarch64 in 8be05b6b7038de7a25fa329f8465bd303cf5afb5